hibernate-core/src/main/java/org/hibernate/type/BasicTypeRegistry.java

```diff
@@ -71,6 +71,7 @@ public BasicTypeRegistry() {
 		register( DbTimestampType.INSTANCE );
 		register( CalendarType.INSTANCE );
 		register( CalendarDateType.INSTANCE );
+		register( CalendarTimeType.INSTANCE );
```

Comment:
The only code change in this PR. From the basic type mapping table, it seems we ignored `CalendarTimeType` by accident.